src/js/App/Header/Tools.js

```diff
@@ -150,26 +149,21 @@ const Tools = () => {
   );
 
   return (
-    <PageHeaderTools widget-type="InsightsToolbar">
-      {/* Show tools on medium and above screens */}
-      <PageHeaderToolsGroup visibility={{ default: 'hidden', lg: 'visible' }}>
+    <div widget-type="InsightsToolbar">
+      <ToolbarItem>
         {isInternal && !window.insights.chrome.isProd && (
-          <PageHeaderToolsItem isSelected={window.insights.chrome.getBundle() === 'internal'}>{<InternalButton />}</PageHeaderToolsItem>
+          <div isSelected={window.insights.chrome.getBundle() === 'internal'}>{<InternalButton />}</div>
```

Comment:
Since we are using just a `div` I don't think we have to pass the `isSelected` prop in here anymore.